client/helpers.js.coffee

```diff
@@ -90,6 +98,29 @@ closedTicketStatuses = ['Closed', 'Delivery QA', 'Deployed', 'Review']
     { fields: { 'points': 0 } }
   ).fetch()
 
+@getAllTimeSpent = ->
+  bugsTime = 0
+  featureTime = 0
+  nonDevTime = 0
+  bugAddition = getAllBugs().forEach((ticket) ->
```

Comment:
You should be able to use reduce for these functions - http://underscorejs.org/#reduce